osu.Game.Rulesets.Osu/UI/OsuPlayfieldAdjustmentContainer.cs

```diff
@@ -55,6 +65,7 @@ protected override void Update()
                 // Scale = 819.2 / 512
                 // Scale = 1.6
                 Scale = new Vector2(Parent.ChildSize.X / OsuPlayfield.BASE_SIZE.X);
+                Position = new Vector2(0, (PlayfieldShift ? 8f : 0f) * Parent.ChildSize.Y / OsuPlayfield.BASE_SIZE.Y);
```

Comment:
Not a huge fan of this being done in Update but I guess it's in line with what is already there.